### ISSUE TITLE
Is it possible to improve online demo performance with any hyperparameter changes?

### ISSUE THREAD

**Issue by JoshMSmith44** [google-deepmind/tapnet#60]
In some point tracking works, for example, temporal window size can be adjusted to improve performance. Are there any hyper-parameter changes from the default live_demo (causal model) that can improve tracking performance, even if it increases model runtime or memory use?

**Comment by cdoersch** [comment]
Sorry I missed this.  

This model is trained end-to-end, so there's no simple way to trade off compute time for performance.  

If your data does not contain any occlusions and objects move slowly, it may be possible to increase accuracy (and compute time) by replacing the TAP-Net-style global search with an initialization that uses the output from the prior frames.  Feeding this directly into the refinement iterations should work fine.  However, this would be a non-trivial code change and not something we directly support.  If you try it, let us know how it goes.

**Comment by cdoersch** [comment]
Closing this due to inactivity.
